aws/structure.go

```diff
@@ -2254,6 +2254,38 @@ func expandConfigRuleSource(configured []interface{}) *configservice.Source {
 	return &source
 }
 
+func expandConfigRemediationConfigurationParameters(configured *schema.Set) map[string]*configservice.RemediationParameterValue {
```

Comment:
Nit: We are preferring to include single resource functions with the resource code, instead of adding them to `aws/structure.go` 👍